etc/development/config-web.toml

```diff
@@ -1,9 +1,10 @@
 [webserver]
-# host = "localhost"
-# port = 16677
+host = "0.0.0.0"
+port = 16677
```

Comment:
Not sure about this. It is needed for your sandbox setup, right?

Reply:
It is needed, otherwise you can't connect to the development server running within Vagrant.